pyproject.toml

```diff
@@ -4,13 +4,14 @@ build-backend = "hatchling.build"
 
 [project]
 name = "tabpfn-client"
-version = "0.0.21"
+version = "0.0.23"
 requires-python = ">=3.10"
 dependencies = [
```

Comment:
these dependencies are out of sync with "requirements.txt" already, which is the risk of having them separately here. 

Please, either don't have requirements.txt and install the project in the package tests directly (`pip install .`) instead of the requirements or use https://github.com/repo-helper/hatch-requirements-txt

Reply:
I went with `hatch-requirements-txt`.
Tested too.